### PR TITLE
Me routes

### PR DESCRIPTION
**Pull request by ndt93**

### CODE REVIEW

File: app_server/app/controllers/SubscriptionController.js [hunk 1]
```diff
@@ -136,6 +141,11 @@ Class.getSubscribers = function(request, reply) {
       return reply(result);
     });
 };
+Class.getSelfSubscribers = function(request, reply) {
+  logger.debug('Get list of self subscribers');
```
Comment (yj-soh) [inline app_server/app/controllers/SubscriptionController.js]: "self subscribers" isn't really too self-explanatory.
Also, add a line break between this and the previous.

File: app_server/test/controllers/SubscriptionController.js [hunk 2]
```diff
@@ -286,6 +287,47 @@ lab.experiment('SubscriptionController Tests', function() {
...
+                     credentials: credentials}, (res) => {
+        Code.expect(res.result).to.have.length(2);
+        Code.expect(res.result[0].username).to.equal(bob.username);
+        Code.expect(res.result[1].username).to.equal(carlos.username);
+        done();
+      });
+    }
+
+    // First subscription
+    Promise.join(userPromise1, userPromise2, userPromise3,
```
Comment (yj-soh) [inline app_server/test/controllers/SubscriptionController.js]: Shouldn't this segment be above the other two functions?

File: app_server/test/controllers/UserController.js [hunk 3]
```diff
@@ -100,6 +101,23 @@ lab.experiment('UserController Tests', {timeout: 5000}, function () {
...
+    Service.createNewUser(bob).then(function (result) {
+      return Service.getUserById(result.userId);
+    }).then(function(user) {
+      let credentials = TestUtils.copyObj(user,
+                                          ['userId', 'username', 'password']);
+      credentials.scope = Authenticator.SCOPE.USER;
+      Router.inject({url: '/api/users/me',
+                     credentials: credentials},
+                    function (res) {
+                      console.log(res.result);
```
Comment (yj-soh) [inline app_server/test/controllers/UserController.js]: You may want to remove this.

**Comment by ndt93** [comment]
Fixed. You can check again

**Comment by yj-soh** [comment]
Looks okay, go ahead.

**Pull request merged**